src/renderer/components/jotz.js

```diff
@@ -3,48 +3,69 @@ var SideMenu = require('./side_menu/side_menu');
 var Content = require('./content/content');
 var TopBar = require('./top_bar/top_bar');
 var actionCreator = require('../actions/action_creator');
+//var NotesStore = require('../stores/notes');
+//var NotebookStore = require('../stores/notebooks');
 
 /*
   This is where the applications state is created/managed.
   TODO: Link up fetchNotes to allNotes for notelist view (getInitialState)
-  TODO: Decide on React.js style guide eg: changeView vs newNote
  */
 
 var Jotz = React.createClass({
 
-  //State created here
   getInitialState: function() {
     return {
+      notes: null,
```

Comment:
since this is now state can we just write: 
`notes: NotebookStore.toJSON()`

When the stores update it should cascade the new state changes down the cascaded props.